apps/server/src/modules/tool/common/domain/error/tool-parameter-unknown.loggable-exception.ts

```diff
@@ -0,0 +1,20 @@
+import { ValidationError } from '@shared/common';
+import { ErrorLogMessage, Loggable, LogMessage, ValidationErrorLogMessage } from '@src/core/logger';
+import { CustomParameterEntry } from '../custom-parameter-entry.do';
+
+export class ToolParameterUnknownLoggableException extends ValidationError implements Loggable {
+	constructor(private readonly parameterEntry: CustomParameterEntry) {
+		super(`tool_param_unknown: The parameter with name ${parameterEntry.name} is not part of this tool.`);
```

Comment:
Please check if the client can handle these changes, because we have a mapping there for the message prefix. 
And if you feel like it, you could create a refactoring ticket if it doesn't exist yet to change the mapping to the type of loggable